app/assets/stylesheets/carstyling.css

```diff
@@ -0,0 +1,61 @@
+.all_cars_container {
+  display: flex;
+  justify-content: space-around;
+  position: relative;
+}
+
+.car_card {
+  box-shadow: 16px 22px 31px -10px rgba(0, 0, 0, 0.7) inset;
+  -webkit-box-shadow: 16px 22px 31px -10px rgba(0, 0, 0, 0.7) inset;
+  -moz-box-shadow: 16px 22px 31px -10px rgba(0, 0, 0, 0.7) inset;
+  border: 0.02rem solid #99c210;
+}
+
+.car_card:hover {
+  background-color: #99c210;
+  transition: all 0.4s ease;
+  transform: perspective(300px) translate3d(10px, 0, 0);
+}
+
+.car_container {
+  width: 45%;
+}
+
+.image {
+  width: 100%;
+  height: 56vh;
+}
+
+.image img {
+  height: 100%;
+  width: 100%;
+}
+
+.name_description {
+  max-width: 30rem;
+  height: 6rem;
+  padding-left: 0.5rem;
+}
+
+.name_description h4 {
+  text-align: center;
+  color: #ffc600;
+  font-size: 1.2rem;
+  font-weight: bold;
+}
+
+.name_description p {
+  text-align: center;
+  color: grey;
+  margin-top: 0.1rem;
+}
+
+@media only screen and (max-width: 600px) {
```

Comment:
- Kindly fix the mobile version styles so that your page displays one car at a time in mobile version.

Reply:
Dear @NATASHA-ct
Depending on the project requirements, we were required to make the mobile responsiveness and there were no specific number of cars in one row.  Kindly, consider the attached requirement for reference.
![Screenshot from 2022-12-14 20-15-05](https://user-images.githubusercontent.com/99394374/207662810-e17ab526-1b28-4282-970d-7a2ceb207995.png)